api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/util/PaymentChannels.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.payment.api.contract.util;
+
+public enum PaymentChannels {
```

Comment:
We already keep all the Payment Channels/Providers/Methods in DB, and access through the models at package uk.gov.hmcts.payment.api.model;
Can we not utilise them here.

Reply:
It seems like we already have some classes for paymentChannel and provider in code. @aravindan89 can we not reuse those classes ?

Reply:
Reused the existing class and deleted the new one.